cmd/cli/update.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	helm "helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/strvals"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openservicemesh/osm/pkg/cli"
+)
+
+const updateDesc = `
+This command updates an osm control plane on the Kubernetes cluster.
+
+The default Kubernetes namespace that gets updated is called osm-system. 
+To update a control plane components in a different namespace, use the 
+--namespace flag.
+
+Example:
+  $ osm update --namespace hello-world --enable-egress false
+`
+
+type updateCmd struct {
+	out       io.Writer
+	clientSet kubernetes.Interface
```

Comment:
I'm not seeing the clientSet being used anywhere in this code

Reply:
Will omit